sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/LiveMetricsRedirectPolicy.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Monitor.OpenTelemetry.AspNetCore.Internals.LiveMetrics
+{
+    internal sealed class LiveMetricsRedirectPolicy : HttpPipelinePolicy
+    {
+        // To prevent circular redirects, max redirect is set to 1.
+        internal const int MaxRedirect = 1;
```

Comment:
Is it always 1, ingestion uses a different value?

Reply:
Ingestion uses 10. Oleg wrote a detailed spec for Ingestion.
LiveMetrics hasn't captured additional details in it's spec. I can double check what AI SDK does.

Reply:
Added tests. https://github.com/Azure/azure-sdk-for-net/pull/44383/commits/62b574f219843376cc75272fa29b9163643fc4a5

Needed to make a few smaller changes to facilitate the test.

Reply:
AI SDK handles only one redirect

Reply:
i'm going to remove the MaxRedirect and while loop since it adds unnecessary complexity.